source/app.d

```diff
@@ -85,13 +85,13 @@ int main(string[] args)
 
 	if (opt.files.length && opt.files.any!(f => !f.exists()))
 	{
-		stderr.writefln!"One or more files specified [%-('%s', %)] do not exist!"(opt.files.filter!(f => !f.exists()));
+		stderr.writefln!"One or more files specified [%(%s, %)] do not exist!"(opt.files.filter!(f => !f.exists()));
```

Comment:
why?

Reply:
Because:
```d
writefln("One or more files specified [%-('%s', %)] do not exist!", ["foo", "bar"]);
```
produce this output:
```
One or more files specified ['foo', 'bar] do not exist!
```

The behaviour of this syntax trims the last char and without the `-` it already encapsulates the string in quotes.

Reply:
To do the behaviour you wanted, you need to write something like this `[%-('%s'%|, %)]`, although I find this unnecessarily unreadable for the current purpose.

Reply:
All error messages output with `''`. If you wish to change it, then change all occurrences.

Reply:
Done, changed the tests too.